src/amqp_channel_sup_sup.erl

```diff
@@ -42,4 +42,4 @@ init([Type, Connection, ConnName]) ->
     {ok, {{simple_one_for_one, 0, 1},
           [{channel_sup,
             {amqp_channel_sup, start_link, [Type, Connection, ConnName]},
-            temporary, brutal_kill, supervisor, [amqp_channel_sup]}]}}.
+            temporary, infinity, supervisor, [amqp_channel_sup]}]}}.
```

Comment:
Disregard my comment. This is a "supervisor supervisor" where calculating the timeout is very hard. `amqp_channel_sup` uses `?WORKER_TIMEOUT` so it be fine to use `infinity` here.